### ISSUE TITLE
BSQL queries 10, 18, 19 fail creating table from text file

### ISSUE THREAD

**Issue by beckernick** [rapidsai/gpu-bdb#200]
These queries create tables from the sentiment text files, but are now throwing.

```
https://github.com/rapidsai/gpu-bdb/blob/89c1f0e643732c0e3d1cceebe96be5838a4f5433/gpu_bdb/queries/q10/gpu_bdb_query_10_sql.py#L91-L97
```
Example error:

```
10
/raid/nicholasb/miniconda3/envs/rapids-gpubdb-20210319/lib/python3.7/site-packages/cudf/core/column/string.py:760: UserWarning: `n` parameter is not supported when `pat` and `repl` are list-like inputs
  "`n` parameter is not supported when "
Encountered Exception while running query
Traceback (most recent call last):
  File "/raid/nicholasb/prod/gpu-bdb/gpu_bdb/bdb_tools/utils.py", line 324, in run_bsql_query
    config=config,
  File "/raid/nicholasb/prod/gpu-bdb/gpu_bdb/bdb_tools/utils.py", line 61, in benchmark
    result = func(*args, **kwargs)
  File "queries/q10/gpu_bdb_query_10_sql.py", line 94, in main
    names="sentiment_word")
  File "/raid/nicholasb/miniconda3/envs/rapids-gpubdb-20210319/lib/python3.7/site-packages/pyblazing/apiv2/context.py", line 2275, in create_table
    + "with all the needed extra parameters. See https://docs.blazingdb.com/docs/creating-tables"
Exception: ERROR: Your input file doesn't have a recognized extension, you have to specify the `file_format` parameter. Recognized extensions are: [orc, parquet, csv, json, psv].
For example if you are using a *.log file you must pass file_format='csv' with all the needed extra parameters. See https://docs.blazingdb.com/docs/creating-tables
```

**Comment by beckernick** [comment]
@Christian8491 is there a recommended action for this?

**Comment by Christian8491** [comment]
Yes, that's correct. Now we should use the `file_format` parameter. 
In this case: ` file_format="csv"`
I will create the respective PR for this @beckernick

**Comment by beckernick** [comment]
Thanks!

**Comment by beckernick** [comment]
Closed by #202
